static/src/stylesheets/layout/nav/_edition-picker-toggle.scss

```diff
@@ -0,0 +1,10 @@
+button {
```

Comment:
Probably don't need to specify the tag name, since the ID below should be specific enough?

Reply:
I'm going to remove the labels, which'll remove the need to have an `id` on the `<button>`, and instead I'll add a unique selector to these buttons, so there'll be no need to specify the tag here.